content/_code-samples/freeze/py/set_global_freeze.py

```diff
@@ -0,0 +1,26 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountSet, AccountSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # Connect to testnet
+
+# Sender wallet object
+sender_wallet = generate_faucet_wallet(client)
+
+# Build accountset transaction to enable global freeze
+accountset = AccountSet(account=sender_wallet.classic_address,
+    set_flag=AccountSetFlag.ASF_GLOBAL_FREEZE)
+
+# Sign and submit transaction
+stxn = safe_sign_and_autofill_transaction(accountset, sender_wallet, client)
```

Comment:
One more print statement with the same idea :)

Reply:
<img width="590" alt="Screenshot 2022-11-17 062006" src="https://user-images.githubusercontent.com/47371105/202369946-8eafe945-b2ba-4d4a-b05c-373fbbbe7363.png">

Same here